apollo-portal/src/test/java/com/ctrip/framework/apollo/portal/util/AuthUserPasswordCheckerTest.java

```diff
@@ -81,4 +80,24 @@ public void testIsWeakPassword() {
     Assert.assertTrue(res.isSuccess());
   }
 
+  @Test
+  public void testIsWeakPassword2() {
```

Comment:
Maybe we can add more test case like return empty list, single element list?

Reply:
Done.